includes/definitions/netspire.yaml

```diff
@@ -0,0 +1,9 @@
+os: netspire
```

Comment:
So netspire appears to be a range of products. Does netspire cover them all (i.e they run the same firmware)?

Reply:
We have two different devices (audio controller and paging station) which run the same firmware. I am not sure if this extends to their entire range. However, their oacommon-mib covers their entire netspire range.